crates/aptos/src/common/init.rs

```diff
@@ -131,33 +141,64 @@ impl CliCommand<()> for InitTool {
             Network::Custom => self.custom_network(&mut profile_config)?,
         }
 
+        // Set the BIP44 index to 0
+        // TODO: Need to update this allow account at different bip44 derivative path index
+        profile_config.bip44_index = if self.from_ledger {
+            Some("0".to_string())
+        } else {
+            None
+        };
+
```

Comment:
Your other PR deals with this indexing issue right, could we do that here too?

Reply:
Yeah agreed, why not making a simple option `--account-index` or similar, for selecting the account index?